lib/k8s-test-framework/src/namespace.rs

```diff
@@ -29,16 +53,38 @@ impl up_down::CommandBuilder for CommandBuilder {
         }
 
         command.stdin(Stdio::null());
-
         command
     }
 }
 
 /// Create a new [`up_down::Manager`] for the specified `namespace` and using
 /// the specified `kubectl_command`.
-pub fn manager(kubectl_command: &str, namespace: &str) -> up_down::Manager<CommandBuilder> {
+pub fn manager(kubectl_command: &str, config: Config) -> up_down::Manager<CommandBuilder> {
     up_down::Manager::new(CommandBuilder {
         kubectl_command: kubectl_command.to_owned(),
-        namespace: namespace.to_owned(),
+        config,
     })
 }
+
+/// Helper to create a Namespace resource during tests
+pub fn make_namespace(name: String, labels: Option<BTreeMap<String, String>>) -> Namespace {
+    match labels {
+        None => Namespace {
+            metadata: ObjectMeta {
+                name: Some(name),
+                ..Default::default()
+            },
+            spec: None,
+            status: None,
+        },
+        Some(labels) => Namespace {
+            metadata: ObjectMeta {
+                name: Some(name),
+                labels,
```

Comment:
I think it would be clearer to get rid of the `match` and instead do this here:

```
labels: labels.unwrap_or_default()
```

Reply:
Bah, I didn't see that there was a default for `labels` 😭